pkg/scheduler/metrics/job.go

```diff
@@ -37,8 +37,34 @@ var (
 			Help:      "Number of retry counts for one job",
 		}, []string{"job_id"},
 	)
+
+	jobCompletedPhaseCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "job_completed_phase_count",
+			Help:      "Number of jobs in completed phase",
+		}, []string{"job_ns"},
+	)
+
+	jobFailedPhaseCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Subsystem: VolcanoNamespace,
+			Name:      "job_failed_phase_count",
+			Help:      "Number of jobs in failed phase",
+		}, []string{"job_ns", "job_id"},
```

Comment:
`[]string{"job_name"}`

Directly use job_name as the label. This job_name represents the combination of <namespace/name>